docker-entrypoint.sh

```
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Generate ENCRYPTION_KEY if not provided by the user
+if [ -z "$ENCRYPTION_KEY" ]; then
+  export ENCRYPTION_KEY=$(head -c 32 /dev/urandom | od -A n -t x1 | tr -d ' \n')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)